### PR TITLE
Add openApp() action

### PR DESCRIPTION
**Pull request by JosephShenton**
Implemented openApp()

### CODE REVIEW

File: actions_std.go [hunk 1]
```diff
@@ -2160,6 +2160,21 @@ func scriptingActions() {
...
+		parameters: []parameterDefinition{
+			{
+				name:      "appIdentifier",
+				validType: String,
+				noMax:     true,
+			},
+		},
+		make: func(args []actionArgument) []plistData {
+			return []plistData{
+				argumentValue("WFSelectedApp", args, 0), // WFAppIdentifier Previously - changed in iOS 15(?)
```
Comment (electrikmilk) [inline actions_std.go:2174]: `WFAppIdentifier` belongs here instead.

`WFSelectedApp` is a dictionary with a `BundleIdentifier` with the same value as `WFAppIdentifier`.

```go
{
	key:      "WFSelectedApp",
	dataType: Dictionary,
	value: []plistData{
		argumentValue("BundleIdentifier", args, 0),
	},
},
```

Please remove the comment.

[Please confirm action parameters with testing before submitting a PR.](https://github.com/electrikmilk/cherri/blob/main/CONTRIBUTING.md)

File: actions_std.go [hunk 2]
```diff
@@ -2160,6 +2160,21 @@ func scriptingActions() {
 			}
 		},
 	}
+	actions["openApp"] = actionDefinition{
+		identifier: "openapp",
```
Comment (electrikmilk) [inline actions_std.go:2164]: If the action key (e.g. `actions["key"]`) and the `identifier` are the same, the identifier field can be removed.

**Review by electrikmilk — CHANGES_REQUESTED** [review]

**Comment by electrikmilk** [comment]
Thanks for making the first contribution!

**Comment by JosephShenton** [comment]
Amazing, fixing these up now!

**Comment by JosephShenton** [comment]
Hey! They should be fixed. I've tried squashing the commits, but I am unsure as to if that succeeded. Please let me know if it didn't.

**Comment by electrikmilk** [comment]
> Hey! They should be fixed. I've tried squashing the commits, but I am unsure as to if that succeeded. Please let me know if it didn't.

The action itself looks great! But, the 7 commits should be squashed into 1 commit.

Please try the following
```
git reset --soft HEAD~1
git push --force
```

**Comment by JosephShenton** [comment]
Hey! Yup! I can do that!

**Comment by JosephShenton** [comment]
Hey! It should be fixed now!

**Comment by electrikmilk** [comment]
> Hey! It should be fixed now!

Thanks for your contribution!

**Pull request merged**